src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/GoalEntityAssert.kt

```diff
@@ -13,11 +13,15 @@ fun assertThat(actual: GoalEntity?) = GoalEntityAssert(actual)
 class GoalEntityAssert(actual: GoalEntity?) :
   AbstractObjectAssert<GoalEntityAssert, GoalEntity?>(actual, GoalEntityAssert::class.java) {
 
+  companion object {
+    private val JPA_MANAGED_FIELDS = arrayOf("id", "createdAt", "createdBy", "createdByDisplayName", "updatedAt", "updatedBy", "updatedByDisplayName")
+  }
+
   fun hasJpaManagedFieldsPopulated(): GoalEntityAssert {
     isNotNull
     with(actual!!) {
-      if (id == null || createdAt == null || createdBy == null || updatedAt == null || updatedBy == null) {
-        failWithMessage("Expected entity to have the JPA managed fields populated, but was [id = $id, createdAt = $createdAt, createdBy = $createdBy, updatedAt = $updatedAt, updatedBy = $updatedBy")
+      if (id == null || createdAt == null || createdBy == null || createdByDisplayName == null || updatedAt == null || updatedBy == null || updatedByDisplayName == null) {
```

Comment:
This method was not including all of the JPA managed fields (we added the 2 new displayNames ones recently, but must have forgotten to update this assertion class! 😬 )